### ISSUE TITLE
[BUG]: The code comment is incorrect

### ISSUE THREAD

**Issue by dohyeonYoon** [serengil/deepface#1401]
### Before You Report a Bug, Please Confirm You Have Done The Following...

- [X] I have updated to the latest version of the packages.
- [X] I have searched for both [existing issues](https://github.com/serengil/deepface/issues) and [closed issues](https://github.com/serengil/deepface/issues?q=is%3Aissue+is%3Aclosed) and found none that matched my issue.

### DeepFace's version

v0.0.94

### Python version

3.10.0

### Operating System

ubuntu 22.04

### Dependencies

absl-py==2.1.0
astunparse==1.6.3
beautifulsoup4==4.12.3
blinker==1.9.0
cachetools==5.5.0
certifi==2024.12.14
charset-normalizer==3.4.0
click==8.1.7
-e git+https://github.com/serengil/deepface.git@a402f09bc8d16ac52dd0d222045c07df021cf8fd#egg=deepface
filelock==3.16.1
fire==0.7.0
Flask==3.1.0
Flask-Cors==5.0.0
flatbuffers==24.3.25
gast==0.6.0
gdown==5.2.0
google-auth==2.37.0
google-auth-oauthlib==1.0.0
google-pasta==0.2.0
grpcio==1.68.1
gunicorn==23.0.0
h5py==3.12.1
idna==3.10
itsdangerous==2.2.0
Jinja2==3.1.4
joblib==1.4.2
keras==2.14.0
libclang==18.1.1
lz4==4.3.3
Markdown==3.7
markdown-it-py==3.0.0
MarkupSafe==3.0.2
mdurl==0.1.2
ml-dtypes==0.2.0
mtcnn==1.0.0
namex==0.0.8
networkx==3.4.2
numpy==1.23.5
oauthlib==3.2.2
opencv-python==4.10.0.84
opt_einsum==3.4.0
optree==0.13.1
packaging==24.2
pandas==2.2.3
pillow==11.0.0
protobuf==4.25.5
pyasn1==0.6.1
pyasn1_modules==0.4.1
Pygments==2.18.0
PySocks==1.7.1
python-dateutil==2.9.0.post0
pytz==2024.2
requests==2.32.3
requests-oauthlib==2.0.0
retina-face==0.0.17
rich==13.9.4
rsa==4.9
scikit-learn==1.6.0
scipy==1.14.1
six==1.17.0
soupsieve==2.6
tensorboard==2.14.1
tensorboard-data-server==0.7.2
tensorflow==2.14.0
tensorflow-estimator==2.14.0
tensorflow-io-gcs-filesystem==0.37.1
termcolor==2.5.0
tf_keras==2.18.0
threadpoolctl==3.5.0
tqdm==4.67.1
typing_extensions==4.12.2
tzdata==2024.2
urllib3==2.2.3
Werkzeug==3.1.3
wrapt==1.14.1

### Reproducible example

```Python
import cv2
import os
import numpy as np
from deepface import DeepFace

def process_image_faces(image_path, output_dir="./output/faces"):
    """Function to detect and save faces from a single image"""
    # Create output directory
    os.makedirs(output_dir, exist_ok=True)
    
    # Read image
    image = cv2.imread(image_path)
    if image is None:
        raise ValueError("Could not read the image.")
    
    try:
        # Face detection
        face_objs = DeepFace.extract_faces(
            img_path=image,
            detector_backend="retinaface",
            enforce_detection=False,
            normalize_face=False
        )
        
        # Save detected faces to files
        for face_idx, face_obj in enumerate(face_objs):
            face_img = face_obj['face']
            
            # Filename format: face_{number}.jpg
            face_filename = os.path.join(output_dir, f"face_{face_idx:02d}.jpg")
            cv2.imwrite(face_filename, face_img)
            
        return face_objs
        
    except Exception as e:
        print(f"Error during face detection: {str(e)}")
        return []

# Usage example
if __name__ == "__main__":
    image_path = "./input/img1.png"  # Modify with actual image file path
    faces = process_image_faces(image_path)
```

### Relevant Log Output

-

### Expected Result

-

### What happened instead?

-

### Additional Info

A face recognition model (e.g., ArcFace) requires an RGB channel image as input. However, upon inspection, the extract_faces() function in deepface/modules/representation.py returns the value of img_objs['face'] as a numpy array in the BGR channel format.
![deepface_bug](https://github.com/user-attachments/assets/b8e957ff-2f06-47bc-ada3-1b02b6e8e26e)

 Therefore, the comment on line 119 of representation.py should be changed from "rgb to bgr" to "bgr to rgb."
https://github.com/serengil/deepface/blob/a402f09bc8d16ac52dd0d222045c07df021cf8fd/deepface/modules/representation.py#L119C1-L119C21

(While the functionality of the code is not affected, the incorrect comment could cause confusion.)
